packages/grid/x-data-grid-premium/LICENSE

```diff
@@ -3,10 +3,10 @@ Commercial License
 Copyright (c) 2020 Material-UI SAS
 
 MUI X Premium (https://mui.com/pricing/) is commercial software. You must purchase
-a license and agree to the End User License Agreement (EULA: https://mui.com/x/license/)
+a license and agree to the End User License Agreement (EULA: https://mui.com/r/x-license-eula)
 to be able to use the software.
 
-Commercial licenses can be obtained at https://mui.com/store/items/material-ui-premium/.
+Commercial licenses can be obtained at https://mui.com/r/x-license.
```

Comment:
For these cases, shouldn't we point to the store or the pricing page?

The pricing page is better for a chance of upselling, which doesn't apply for premium, but if we're planning to move to a single page (before the cart page) in the near future, it makes sense to set it as the single URL already.

Reply:
I'm not following, this link points to the store.

Reply:
Sorry, my mistake, I had in mind [we'd update the r-license link](https://github.com/mui/material-ui/pull/32464#discussion_r858801076) and I had missed the _redirects file on this PR.

But in any case, perhaps we should send them to the pricing page, where the customers can see their licensing options. WDYT?
With the current link, a potential Premium user will be redirected to the Pro store page, which besides being confusing, there's no easy flow to the Premium page (we could add a link to mitigate this part, but it's not the ideal solution).

Reply:
It's an interesting point on being able to differentiate pro from premium users. I have pushed a commit to handle the problem 👍